src/components/molecules/VenueChat/VenueChat.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useContext, useEffect, useState, useMemo, FC } from "react";
+import { ChatContext } from "components/context/ChatContext";
+import "./CampChat.scss";
+import { useUser } from "hooks/useUser";
+import { useSelector } from "hooks/useSelector";
+import { useVenueId } from "hooks/useVenueId";
+import { getDaysAgoInSeconds } from "utils/time";
+import { VENUE_CHAT_AGE_DAYS } from "settings";
+import ChatBox from "components/molecules/Chatbox";
+import useRoles from "hooks/useRoles";
+
+interface ChatOutDataType {
+  messageToTheBand: string;
+}
+
+const VenueChat: FC = () => {
+  const venueId = useVenueId();
+  const { userRoles } = useRoles();
+  const { user } = useUser();
+  const { chats, venue } = useSelector((state) => ({
+    chats: state.firestore.ordered.venueChats,
+    venue: state.firestore.data.currentVenue,
+  }));
+
+  const [isMessageToTheBarSent, setIsMessageToTheBarSent] = useState(false);
+
+  useEffect(() => {
+    if (isMessageToTheBarSent) {
+      setTimeout(() => {
+        setIsMessageToTheBarSent(false);
+      }, 2000);
+    }
+  }, [isMessageToTheBarSent]);
```

Comment:
[NB] I've seen this general timeout/boolean pattern a couple of times in this PR. I wonder if it would be worth extracting it into a more generic hook?